src/Pages/HistoryPage/HistoryPage.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { useHistoryContext } from '../../Contexts/History.context';
+import { getVideoListById } from '../../Api functions/getVideosById';
+import { Header } from '../../Components/Header/Header';
+import { SideBar } from '../../Components/SideBar/SideBar';
+import { PlayListContainer } from '../../Components/PlayListContainer/PlayListContainer';
+
+export function HistoryPage() {
+  const [historyVideo, setHistory] = React.useState([]);
+  const [state, dispatch] = useHistoryContext();
+  const [loading, setLoading] = React.useState(true);
+  console.log(state);
+  React.useEffect(() => {
+    setLoading(true);
+    getVideoListById(state).then((data) => {
+      setHistory(data);
+      setLoading(false);
```

Comment:
I hope you will write this using async-await later on.

Reply:
yes